c/src/ml-api-common.c

```diff
@@ -1325,3 +1325,61 @@ ml_option_set (ml_option_h option, const char *key, void *value,
 
   return ML_ERROR_NONE;
 }
+
+/**
+ * @brief Gets a value of key in ml-info instance.
+ */
+int
+ml_info_get (ml_info_h info, const char *key, char **value)
+{
+  ml_info_s *_info;
+  gchar *_value;
+
+  check_feature_state (ML_FEATURE);
+
+  if (!info) {
+    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
+        "The parameter, 'info' is NULL. It should be a valid ml_info_h");
+  }
+
+  if (!key) {
+    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
+        "The parameter, 'key' is NULL. It should be a valid const char*");
+  }
+
+  if (!value) {
+    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
+        "The parameter, 'value' is NULL. It should be a valid char**");
+  }
+
+  _info = (ml_info_s *) info;
+  _value = (gchar *) g_hash_table_lookup (_info->info_table, key);
```

Comment:
let's consider to check valid key string,
```
if (!g_hash_table_contains (key)) 
  _ml_error_report_return (ML_ERROR_INVALID_PARAMETER, ...
```

Reply:
I added those lines, thanks.